src/Entity/ApiDoc.php

```diff
@@ -179,6 +223,7 @@ public static function baseFieldDefinitions(EntityTypeInterface $entity_type) {
     $fields['spec'] = BaseFieldDefinition::create('file')
       ->setLabel('OpenAPI specification')
```

Comment:
Just noticed that these need t(), but we could handle in another ticket

Reply:
added #17

Reply:
Easy to fix.